### DISCUSSION TITLE
你好，请问有没有编译教程，或者能不能加入1.8.4版本

### DISCUSSION THREAD

**Discussion by wx1141235171** [querycap/istio#59] [Q&A] [answered]
由于需要使用1.8.4版本，而querycap的docker库里没有此版本，想尝试自己编译，但是在编译proxy期间报错，求帮助

**Comment by morlay** [comment, marked as answer]
版本太老了，没精力维护

参考这里的依赖需求 https://github.com/querycap/istio/blob/master/build/istio-envoy-build-env/Dockerfile.istio-envoy-build-env
把编译环境的依赖的版本了和这里对应 https://github.com/istio/tools/blob/release-1.8/docker/build-tools/Dockerfile#L552
构建编译环境

或者你可以试试用 https://hub.docker.com/r/querycapistio/istio-envoy-build-env/tags?page=1&ordering=last_updated&name=release-1.9 （1.8 到 1.9 编译环境貌似没太大变化）

至少 12u 的 aarch64 服务器（鲲鹏 1 个多小时），用 1.8 的编译环境 https://github.com/querycap/istio/blob/master/build/istio/Dockerfile.istio-envoy#L10，先把 envoy 编译出来

最后替换 https://github.com/querycap/istio/blob/master/build/istio/Dockerfile.proxyv2#L42
